src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
```

Comment:
in this case you aren't exporting any functions so these should be camelCase please.

Reply:
preferred syntax is also

```js
const menuAction = (elm) => {
  // code goes here
};
```

Reply:
done